tap_exacttarget/endpoints/campaigns.py

```diff
@@ -44,7 +45,9 @@ def sync_data(self):
             FuelSDK.ET_Campaign,
             self.auth_stub)
 
+        catalog_copy = copy.deepcopy(self.catalog)
+
         for campaign in cursor:
             campaign = self.filter_keys_and_parse(campaign)
 
-            singer.write_records(self.__class__.TABLE, [campaign])
+            self.write_records_with_transform(campaign, catalog_copy, self.__class__.TABLE)
```

Comment:
Can we change the `self.__class__.TABLE` to `self.TABLE`?

Reply:
Done the changes.